### ISSUE TITLE
ERROR - Navigation error: the JSON object must be str, bytes or bytearray, not NoneType

### ISSUE THREAD

**Issue by shadlia** [lavague-ai/LaVague#429]
**Describe the bug**
When attempting to log the LLM output using the Llama3 model locally, I encounter the following `error: ERROR - Navigation error: the JSON object must be str, bytes or bytearray, not NoneType. `The logged output appears as shown in the screenshot.

![llama fail](https://github.com/lavague-ai/LaVague/assets/74935083/7e530f81-24da-4491-9511-fa151e6889e5)

**Expected behavior**
I expected the output to be correctly logged, similar to the results obtained when testing with the Gemini model, without encountering the JSON object error.
![gemini success text](https://github.com/lavague-ai/LaVague/assets/74935083/453758ec-df28-4bfe-b99d-dae12e4ac69e)

**Comment by dhuynh95** [comment]
Llama 3 might not perform well enough and the extraction pipeline might fail. It also might happen because ollama quantizes things and the output is a bit random.

Could you try with Groq?

**Comment by shadlia** [comment]
i tried with Groq and after some prompt adjustments it worked ! thank u ! i was using llama 3 8B and when i tested with Groq i used Llama 3 70B , could it be one of the reasons ?

**Comment by dhuynh95** [comment]
Yes, Llama 3 70b is much better than 8b version. We did some testing and could observe that:

![image](https://github.com/lavague-ai/LaVague/assets/36925557/f448b55f-00ed-42e0-9e59-2309c0d44995)

You can find more on our [blog article on our Web Agent evaluation](https://blog.lavague.ai/evaluating-open-source-and-closed-models-for-web-navigation/).
